examples/resources/nxos_bgp_peer/resource.tf

```diff
@@ -7,4 +7,6 @@ resource "nxos_bgp_peer" "example" {
   peer_template    = "SPINE-PEERS"
   peer_type        = "fabric-internal"
   source_interface = "lo0"
+  hold_time        = 5
+  keep_alive       = 15
```

Comment:
I think you need to run `go generate` again to update the code and documentation accordingly.